### ISSUE TITLE
Add a `setup` function to card bps

### ISSUE THREAD

**Issue by IsakSkole123** [LunarTides/Hearthstone.js#159]
Add a `setup` function that runs in the constructor code of the card. This function can then do something like this
```js
setup(plr, game, self) {
    self.addKeyword("Divine Shield");
    self.addKeyword("Taunt");
}
```
instead of
```js
keywords: ["Divine Shield", "Taunt"],
````
in the card bp.

Also remove the `keywords`, `corrupt`, `colossal`, etc... from the bp and instead do all that in the setup function

- [x] Create the setup function and make it work
- [x] Remove the fields from the bps
- [x] Update all the cards
- [x] Update the example cards

**Comment by IsakSkole123** [comment]
I changed my mind, this is just going to complicate things. Maybe reopen this in the future

**Comment by IsakSkole123** [comment]
Reopening this, but won't give it a milestone. Do this whenever.

**Comment by LunarTides** [comment]
This will be a problem if, for example, `Battlecry: Discover a card with Divine Shield`.

**Comment by LunarTides** [comment]
How about in `getCards`:
```ts
// return cards.filter(c => !c.uncollectible || !uncollectible);
return cards.map(c => new game.Card(c. plr)).filter(c => !c.uncollectible || !uncollectible);
```

**Comment by LunarTides** [comment]
Closed by dedc2d5
